### DISCUSSION TITLE
Domen РФ

### DISCUSSION THREAD

**Discussion by an-140** [uBlockOrigin/uAssets#20025] [Q&A]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

I can't block domen "рф" (Russian Federation) in results of search system

### URL(s) where the issue occurs.

`https://магнитогорск.добрострой.рф/products/svetilnik-svetodiodnyy-potolochnyy-acl-106-90w-kvadrat-belyy/`

### uBO version

uBlock Origin 1.52.2

### Browser name and version

 Chrome 117.0.5938.150

### Settings

*://toolb.ru/*
*://www.neometer.ru/*
*://*.ru/*
*://*.by/*
*://*.domotehnika.store/*
*://*.all.biz/*
*://*.instrumentall.md/*
*://*.delta.kg/*
*://*.rusarmy.com/*
*://*.eti.su/*
*://*.dak-s.com/*
*://*.рф/*

**Comment by dportvine** [comment]
`punycode: .xn--p1ai` = `.рф` , try this - `://*.xn--p1ai`

**Comment by ghost** [comment]
Put just `рф` into "My filters" and it will work.

![image](https://github.com/uBlockOrigin/uAssets/assets/886325/bc023d86-987d-4caa-aed6-129932f71f94)


![image](https://github.com/uBlockOrigin/uAssets/assets/886325/5a5aa627-ee60-4cea-98d6-b8b08c3a042d)

**Reply by krystian3w** [reply]
So now possible ban new reddit by:
```adb
||reddit.com^$doc
@@||old.reddi.com^$doc
```

In the past we must skip "`||`".
